site/src/Librecores/ProjectRepoBundle/Controller/ProjectController.php

```diff
@@ -26,45 +25,60 @@ public function newAction(Request $request)
-            'choices' => $parentChoices,
-            'choices_as_values' => true,
-            'multiple' => false,
-        ))
-        ->add('name')
-        ->add('sourceRepo', SourceRepoType::class)
-        ->add('save', SubmitType::class, array('label' => 'Create Project'))
-        ->getForm();
+                     ->add('parentName', ChoiceType::class, array(
+                        'mapped' => false,
+                        'choices' => $parentChoices,
+                        'choices_as_values' => true,
+                        'multiple' => false,
+                     ))
+                     ->add('name')
+                     ->add('sourceRepo', SourceRepoType::class)
+                     ->add('save', SubmitType::class, array('label' => 'Create Project'))
+                     ->getForm();
 
         $form->handleRequest($request);
 
         // save project and redirect to project page
         if ($form->isValid()) {
-            /* XXX: currently only the user namespace is supported, see above.
-             // set parent (extract from string selection box)
-             $formParent = $form->get('parentName')->getData();
-             if (!preg_match('/^[uo]_.+$/', $formParent)) {
-             throw new \Exception("form manipulated");
-             }
-             list($formParentType, $formParentName) = explode('_', $formParent, 2);
-             if ($formParentType == 'u') {
-             $userManager = $this->container->get('fos_user.user_manager');
-             $user = $userManager->findUserByUsername($formParentName);
-             if (null === $user) {
-             throw new \Exception("form manipulated");
-             }
-             $p->setParentUser($user);
-             } else if ($formParentType == 'o') {
-             // TODO: Add ability to add projects to organizations here
-             throw new \Exception("adding projects to organizations is currently not supported.");
-             }*/
+            // set parent (extract from string selection box)
+            $formParent = $form->get('parentName')->getData();
+
+            if (!preg_match('/^[uo]_.+$/', $formParent))
+                throw new \Exception("form manipulated");
+
+            list($formParentType, $formParentName) = explode('_', $formParent, 2);
+
+            if ($formParentType === 'u') {
+                $user = $this->container->get('fos_user.user_manager')
+                                        ->findUserByUsername($formParentName);
+
+                if (null === $user)
+                    throw new \Exception("form manipulated");
+
+                $p->setParentUser($user);
+
+            } else if ($formParentType === 'o') {
+                $organization = $this->getDoctrine()
+                                     ->getRepository('LibrecoresProjectRepoBundle:Organization')
+                                     ->findOneByName($formParentName);
+
+                $p->setParentOrganization($organization);
+            }
```

Comment:
All of this is still hacky as hell, but let's leave it for now. I'll open a follow-up bug to get this fixed properly (doesn't seem to be straightforward though).